app/scripts/modules/core/src/serverGroup/serverGroupSearchResultType.tsx

```diff
@@ -46,22 +54,127 @@ const SearchResultsHeader: SearchResultsHeaderComponent = () => (
     <HeaderCell col={cols.ACCOUNT}/>
     <HeaderCell col={cols.REGION}/>
     <HeaderCell col={cols.EMAIL}/>
+    <HeaderCell col={cols.HEALTH}/>
   </TableHeader>
 );
 
-const SearchResultsData: SearchResultsDataComponent = ({ results }) => (
+const SearchResultsData: SearchResultsDataComponent<IServerGroupSearchResult> = ({ results }) => (
   <TableBody>
-    {results.slice().sort(itemSortFn).map(item => (
+    {results.map(item => (
       <TableRow key={itemKeyFn(item)}>
         <HrefCell item={item} col={cols.SERVERGROUP} />
         <AccountCell item={item} col={cols.ACCOUNT} />
         <BasicCell item={item} col={cols.REGION} />
         <BasicCell item={item} col={cols.EMAIL} />
+        <HealthCountsCell item={item} col={cols.HEALTH} />
       </TableRow>
     ))}
   </TableBody>
 );
 
+interface IServerGroupDataProps {
+  type: ISearchResultType;
+  results: IServerGroupSearchResult[],
+}
+
+interface IServerGroupDataState {
+  serverGroups: IServerGroupSearchResult[],
+}
+
+interface IServerGroupTuple {
+  toFetch: IServerGroupSearchResult;
+  fetched: IServerGroup;
+}
+
+const makeServerGroupTuples = (sgToFetch: IServerGroupSearchResult[], fetched: IServerGroup[]): IServerGroupTuple[] => {
+  const findFetchedValue = (toFetch: IServerGroupSearchResult) => fetched.find(sg => sg.name === toFetch.serverGroup);
```

Comment:
Do you want to match on account and region here, too?

Reply:
yes, I was just seeing if you were paying attention